tests/test.cpp

```diff
@@ -1227,7 +1227,7 @@ void ErrorTest() {
   TestError("enum X:float {}", "underlying");
   TestError("enum X:byte { Y, Y }", "value already");
   TestError("enum X:byte { Y=2, Z=1 }", "ascending");
-  TestError("union X { Y = 256 }", "must fit");
+  TestError("union X { Y = 256 }", "enum value does not fit");
```

Comment:
I don't understand this test.
This declaration `union Any { Monster, Weapon, Pickup }` is correct.
But `union X { Y = 256 }` is incorrect. Here we refer to undefined type `256` not number.

Reply:
AInUnionAgain:A = 10 is not equal to AInUnionAgain = 10 from a sematic point of view.
In any case, this is a minor issue, I think this will be resolved automatically.
Extra check if (ev.value < 0 || ev.value >= 256) will be redurant.